internal/test/compilecheck/compile_check_test.go

```diff
@@ -43,18 +63,17 @@ func TestCompileCheck(t *testing.T) {
 			t.Parallel()
 
 			req := testcontainers.ContainerRequest{
-				Image: image,
-				Cmd:   []string{"tail", "-f", "/dev/null"},
-				Mounts: []testcontainers.ContainerMount{
-					testcontainers.BindMount(rootDir, "/workspace"),
+				Image:      image,
+				Cmd:        []string{"tail", "-f", "/dev/null"},
```

Comment:
[nit] Can we add a note about what this does? Something like "keep container running to Exec commands into it"?